packages/uswds/src/Templates/ButtonTemplates/IconButton.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ComponentProps } from "react";
+import { IconButtonProps as RjsfIconButtonProps } from "@rjsf/utils";
+
+const ICON_URL_BASE = "https://cdn.jsdelivr.net/npm/uswds/src/img/usa-icons/";
+
+// Map icon names to their corresponding SVG filenames and default alt text
+const iconMap: {
+  [key: string]: { filename: string; alt: string };
+} = {
+  add: { filename: "add.svg", alt: "Add Item" },
+  remove: { filename: "remove.svg", alt: "Remove Item" }, // Or close.svg
+  "arrow-upward": { filename: "arrow_upward.svg", alt: "Move Item Up" },
+  "arrow-downward": { filename: "arrow_downward.svg", alt: "Move Item Down" },
+  copy: { filename: "content_copy.svg", alt: "Copy Item" }, // Assuming content_copy.svg exists
+  // Add other icons as needed
+};
+
+// Define our component's props, extending RJSF's IconButtonProps
+// and adding our specific props like iconName and uswdsStyle
+interface IconButtonProps extends RjsfIconButtonProps {
+  iconName?: keyof typeof iconMap; // Make iconName optional, button might just have text
+  uswdsStyle?: "outline" | "unstyled" | "secondary" | "accent-cool" | "base"; // Add more styles as needed
+  children?: React.ReactNode; // Allow passing text content like "Add Item"
+}
+
+const IconButton = ({
+  iconName,
+  uswdsStyle,
+  className,
+  onClick,
+  disabled,
+  title,
+  registry, // Included but not used directly in this basic version
+  uiSchema, // Included but not used directly in this basic version
+  children,
+  icon, // Destructure RJSF's icon/iconType to prevent passing them down
+  iconType,
+  ...props // Capture rest of the props like style
+}: IconButtonProps) => {
```

Comment:
```suggestion
function IconButton({
  iconName,
  uswdsStyle,
  className,
  onClick,
  disabled,
  title,
  registry, // Included but not used directly in this basic version
  uiSchema, // Included but not used directly in this basic version
  children,
  icon, // Destructure RJSF's icon/iconType to prevent passing them down
  iconType,
  ...props // Capture rest of the props like style
}: IconButtonProps) {
```